kratos/solving_strategies/builder_and_solvers/residualbased_block_builder_and_solver.h

```diff
@@ -995,6 +1042,9 @@ class ResidualBasedBlockBuilderAndSolver
     std::vector<IndexType> mSlaveIds;  /// The equation ids of the slaves
     std::vector<IndexType> mMasterIds; /// The equation ids of the master
     std::unordered_set<IndexType> mInactiveSlaveDofs; /// The set containing the inactive slave dofs
+    double mScaleFactor = 1.0;         /// The manuallyset scale factor

```

Comment:
why a class variable?

Reply:
anyway, you are overwriting in all the cases, isn't it?

Reply:
Yes